proto/cork/v1/query.proto

```diff
@@ -69,4 +77,20 @@ message QueryCellarIDsRequest {}
 // QueryCellarIDsResponse is the response type for Query/QueryCellars gRPC method.
 message QueryCellarIDsResponse {
   repeated string cellar_ids = 1;
+}
+
+// QueryScheduledCorksRequest
+message QueryScheduledCorksRequest {}
+
+// QueryScheduledCorksResponse
+message QueryScheduledCorksResponse {
+  repeated ScheduledCork corks = 1;
+}
+
+// QueryScheduledBlockHeightsRequest
+message QueryScheduledBlockHeightsRequest {}
+
+// QueryScheduledBlockHeightsResponse
+message QueryScheduledBlockHeightsResponse {
+  repeated uint64 block_heights = 1;
```

Comment:
In order for this output to be useful, should we include a query for scheduled corks that includes specifying the block height?

Reply:
Added it